gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/GradleCacheExplorer.java

```diff
@@ -57,17 +59,36 @@ public final Set<String> getCompletions(Set<String> repoUrls, DependencyGroup in
                 .flatMap(Optional::stream)
                 .collect(Collectors.toSet());
 
+        Set<String> resultsWithStarsIncluded = results.stream()
+                .map(result -> {
+                    int colonIndex = result.indexOf(':');
+                    if (colonIndex != -1) {
+                        return Arrays.asList(result, result.substring(0, colonIndex) + ":*");
+                    }
+                    return Collections.singletonList(result);
+                })
+                .flatMap(Collection::stream)
```

Comment:
you can replace the collection construction/instantiation with just a `Stream` instead and do a `flatMap` instead i.e.:

```java
.flatMap(result -> {
  int colonIndex = ...
  if (...) {
    return Stream.of(result, result.substring(..));
  }
  return Stream.of(result);
```

Reply:
good idea, I've also added a check to ensure that the count of packages is more than one as we dont want to suggest * for package name with only one option